recipes/r-hash/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: r-hash
+  version: "2.2.6"
+
+source:
+  fn: hash_2.2.6.tar.gz
+  url: http://cran.r-project.org/src/contrib/hash_2.2.6.tar.gz
+  sha256: df93f119d62d4e0d33afcfb5bfd5770565b5a66d766dfb07126b445b572a7d56
+
+build:
+  skip: true  # [win]
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build: 
+    - r-base
```

Comment:
I think that you need to add:
```yaml
    - gcc
```
to fix https://travis-ci.org/conda-forge/staged-recipes/builds/262158689 .